src/main/resources/com/cloudbees/plugins/credentials/ViewCredentialsAction/index.properties

```diff
@@ -1,2 +1,2 @@
 global=(global)
-storesScopedTo=Stores scoped to <a href=".." class='model-link inside'>{0}</a>
+storesScopedTo=Stores scoped to {0}
```

Comment:
Why?

Reply:
It just didn't seem useful, why would you want that there?

I looked at renaming this to be clearer what it was but I didn't come up with anything

Reply:
Good point, I missed that this only ever links to the current object, not to its ancestors (which might have been useful).